### PR TITLE
Completed Basic Grids

### PR DESCRIPTION
**Pull request by varun1233**
1. Added grid 2 layout
2. Added grid 3 layout

### CODE REVIEW

**Review by parvez-rashid — APPROVED** [review]
UI Review: UI looks fantastic, I like the color scheme especially.

As for the code, I couldn't find any irregularities or need for suggestions, great work! Code can be merged.

File: components/grids/grid.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+          <p>
+            Use grid when you want sections in view. Check below some examples.
+          </p>
+        </div>
+        <!-- Component Usage Explanation -->
+        <div class="component-usage-explain">
+          <h2>Example of grid 2 layout</h2>
+        </div>
+        <!-- Component Display -->
+        <div class="component-display justify-content-space-evenly">
```
Comment (eakansh125) [inline components/grids/grid.html:103]: Just a suggestion: class names can be shortened. Like justify-content-space-evenly can be justify-evenly which is equally understandable.
No mandates, we're good here!

Comment (varun1233) [reply components/grids/grid.html:103]: Thanks for the suggestion, will look into this while refactoring after approval.

**Review by eakansh125 — APPROVED** [review]
Clean code, neat UI, no syntactic issues found.
Good to be merged from my end.

**Pull request merged**